### ISSUE TITLE
Add a callback when a Heartbeat message is received

### ISSUE THREAD

**Issue by philips77** [NordicSemiconductor/IOS-nRF-Mesh-Library#471]
Currently, the heartbeats need to be polled to get the heartbeat value. It would be nice to get that information as a callback.
Also, the poll allows only to get approximate value, not exact, as far as I remember.

This can be done together with #470.

**Comment by philips77** [comment]
CC: #469

**Comment by Ahmadshoh** [comment]
Hi. Is there any update here?

**Comment by philips77** [comment]
No, this task will have to wait. I'm working on adding some basic UI for Light LC Server model.

**Comment by philips77** [comment]
Fixed by #705.
